m2m-transfers/m2m-transfers-json-contract.md

```diff
@@ -0,0 +1,196 @@
+#JSON contract for BANNO.M2MTRANSFERS.V1.POW
+
+## GET PRELOADDATA STATE
+### Client Request (PRELOADDATA)
+
+```json
+{
+  "rgState": "PRELOADDATA",
+  "powerOnFileName": "BANNO.M2MTRANSFERS.V1.POW",
+  "userChrList": [
+    { "id": 1, "value": "1234567890" }, // 10-digit account
+    { "id": 2, "value": "" }, // userchar[2-5] is unused
+    { "id": 3, "value": "" },
+    { "id": 4, "value": "" },
+    { "id": 5, "value": "" }
+  ],
+  "userNumList": [], //  usernum[1-5] is unused
+  "rgSession": 1
+}
+```
+
+### PowerOn Successful Response (PRELOADDATA)
+list of institution and member limits, eligible shares, list of scheduled transfers, list of saved recipients
+
+```json
+{
+	"currentState": {
+		"enforceLimits": true,
+		"transferLimits": [{
+			"institutionCount": "5",
+			"institutionAmount": "10000.00",
+			"memberLimitCount": "3",
+			"memberLimitAmount": "5000.00",
+			"memberActualCount": "1",
+			"memberActualAmount": "500.00"
+		}],
+		"availableShares": [{
+				"transferSLId": "0000800005S0002",
+				"transferName": "SUMMER SAVER",
+				"available": "50.00"
+			},
+			{
+				"transferSLId": "0000800005S0010",
+				"transferName": "REGULAR CHECKING",
+				"available": "500.00"
+			},
+			{
+				"transferSLId": "0000800005S0015",
+				"transferName": "ULTIMATE CHECKING (10)",
+				"available": "0.00"
+			}
+		],
+
+		"scheduledTransfers": [{
+				"trnsferLoc": "215",
```

Comment:
what is `trnsferLoc`?

Reply:
typo - `trnsferLoc` should be `transferLoc`

Reply:
Type-o aside, it's the system assigned unique serial number for that particular record - allowing for a definitive way to target the correct record for editing or deleting.